src/api/spotify/service.ts

```diff
@@ -3,21 +3,26 @@ import type { Song, Artist, LocalStorage } from '../spotify/types';
 import fetch from 'cross-fetch';
 
 export async function getUserData() {
-	const accessToken = localStorage.getItem('access_token');
-	const response = await fetch('https://api.spotify.com/v1/me', {
-		headers: {
-			'Content-Type': 'application/json',
-			Authorization: `Bearer ${accessToken}`
-		}
-	});
-	if (!response.ok) {
-		console.log('getUserData error');
-	}
-	const responseData = await response.json();
 	const data = {
-		display_name: responseData.display_name,
-		user_id: responseData.id
+		display_name: '',
+		user_id: ''
 	};
+	const storedValue: string | null = localStorage.getItem('ViBE');
+	if (storedValue !== null) {
+		const obj: LocalStorage = JSON.parse(storedValue);
+		const accessToken = obj.access_token;
+		const response = await fetch('https://api.spotify.com/v1/me', {
+			headers: {
+				'Content-Type': 'application/json',
+				Authorization: `Bearer ${accessToken}`
+			}
+		});
+		if (!response.ok) {
+			console.log('getUserData error');
+		}
+		const responseData = await response.json();
+		(data.display_name = responseData.display_name), (data.user_id = responseData.id);
```

Comment:
whats this syntax mean?

Reply:
will update. I think there was a typo then prettier formatted to that.